### PR TITLE
UI Changes Pull Request - See Dec 4/5th pushes for specific files to comment on

### PR DESCRIPTION
**Pull request by ReedMcCullough**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,1026 @@
...
+  font-size: 1rem;
+  text-transform: none;
+  padding: 0.75rem 1.25rem;
+  margin: 0 0 0.5rem 0;
+  vertical-align: middle;
+  text-align: center;
+  cursor: pointer;
+  text-decoration: none;
+  line-height: 1;
+}
```
Comment (LukeWattsCP) [inline index.css]: The buttons look good on the test version. Could possibly use some comments in this section to explain which buttons are which, but everything looks well designed. Nice job!

File: src/index.css [hunk 2]
```diff
@@ -378,7 +378,7 @@ html {
 
 body {
   color: #404040;
-  background: white;
+  background: rgb(221, 228, 247);
```
Comment (LukeWattsCP) [inline src/index.css]: Good color selection, page is looking pretty

File: src/pages/Subpage.css [hunk 3]
```diff
@@ -42,4 +42,11 @@
     background: #8d8d8d;
     color: #ffffff;
     text-decoration: none;
+  }
+
+  .button_playlist, [type=submit]:focus, [type=reset]:hover, [type=submit]:active, [type=reset]:focus, [type=reset]:active, [type=button]:focus, [type=button]:active {
+    border: 1px solid #000000;
+    background: #585858;
+    color: #ffffff;
+    text-decoration: none;
```
Comment (hayahann) [inline src/pages/Subpage.css:86]: The pages are starting to look visually pleasing! Everything so far looks good to me. The only thing I would add is maybe use colors that create more contrast for emphasis.

File: src/pages/AddSongForm.js [hunk 4]
```diff
@@ -78,10 +87,15 @@ function AddSongForm(props) {
...
                 value={query.val}
                 onChange={handleChange}/>
-            <input type="button" value="Search" onClick={submitForm} />
-            <ResultsHeader id="resultsHeader"/>
+            <input type="button" value="Search" className="button_playlist" onClick={submitForm} />
+            <br></br><br></br>
+            { (afterSearch === false) ? (
+                <ResultsHeader id="resultsHeader"/>
+                ) : (<div/>
+            )}
```
Comment (heberman) [inline src/pages/AddSongForm.js:101]: good, clean solution to only showing this element when afterSearch is false

File: index.css (hunk 1 above)
Comment (ReedMcCullough) [reply index.css]: Very true. I had to experiment on a lot of stuff to see which line does what so comments would be a great change. Thanks!

File: src/index.css (hunk 2 above)
Comment (ReedMcCullough) [reply src/index.css]: Thanks. It took a while to find a color scheme that worked well together

File: src/pages/AddSongForm.js (hunk 4 above)
Comment (ReedMcCullough) [reply src/pages/AddSongForm.js:101]: I agree. I was impressed when this efficient solution actually worked. Looks great on our website too.

File: src/pages/Subpage.css (hunk 3 above)
Comment (ReedMcCullough) [reply src/pages/Subpage.css:86]: Thanks! I agree. I could definitely add some more contrast.

File: src/pages/HomePage.js [hunk 5]
```diff
@@ -50,7 +50,7 @@ function HomePage() {
 
     return (
         <><center>
-            <div id="top"></div>
+            <div id="top"/>
 
             <br></br>
```
Comment (hayahann) [inline src/pages/HomePage.js]: Great Job with the logo! It really improves the appearance of our overall project.

Comment (ReedMcCullough) [reply src/pages/HomePage.js]: I'm glad we came up with the idea for a logo; definitely makes it look more professional

**Comment by ReedMcCullough** [comment]
closed. finished reviews

**Pull request closed without merging**